src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx

```diff
@@ -97,9 +131,7 @@ The APM [Summary](/docs/apm/applications-menu/monitoring/apm-overview-page) page
 * CPU usage
 * Memory
 
-You can toggle between a table view or breakout metric details for the individual hosts by selecting **View table** or **Break out each metric by host**.
-
-For more information on host data on the APM **Summary** page, see [host details](/docs/apm/applications-menu/monitoring/apm-overview-page#host_details).
+To investigate a specific host, click its name and it will reveal an expanded view on the right-hand side of the page.
 
 ## Troubleshoot missing APM data [#troubleshooting]
```

Comment:
I think with what you have at top with requirements, you can delete this section now, as it's already covered.